src/main/kotlin/gg/skytils/skytilsmod/features/impl/spidersden/RainTimer.kt

```diff
@@ -28,18 +28,20 @@ import gg.skytils.skytilsmod.utils.graphics.colors.CommonColors
  * @link https://github.com/PikaFan123/rain-timer
  */
 object RainTimer {
-    var nextRain = 1596552707000
+    const val nextRain = 1727548500000 // Unix-Time of a past rain event start in milliseconds
```

Comment:
```suggestion
    var nextRain = 1727548500000 // Unix-Time of a past rain event start in milliseconds
```

This is not a constant because the value changes on line 37

Reply:
just noticed it too, reverted that change for line 31. Sorry for that, I'm still new to programming but open to learning, thanks.